src/Header.jsx

```diff
@@ -0,0 +1,37 @@
+import {
+    Grid,
+    makeStyles,
+    Tooltip,
+    Typography
+} from '@material-ui/core';
+import InfoIcon from '@material-ui/icons/Info';
+import ReactDOM from "react-dom";
+import React from "react";
+
+const useTooltipStyles = makeStyles(theme => ({
+    tooltip: {
+        color: "white",
+        backgroundColor: "black",
+        fontSize: 14
+    }
+}));
+
+export default function Header (props) {
+    return (
+        <Grid container>
+            <Grid item xs={"auto"}>
+                <Typography variant={window.innerWidth >=380 && "h3" || window.innerWidth >= 280 && "h4" || "h5"} component="h1" gutterBottom>{props.text}</Typography>
```

Comment:
Changing the variant like this seems a bit fishy to me. It's not just the font size that changes, but also things like line height and letter spacing in the default theme. And with a custom theme, other properties such as font family, variant, or color may change.

If you only want to change the font size, then I recommend using a custom class with [responsive font sizes](https://material-ui.com/customization/typography/#responsive-font-sizes) using [breakpoints](https://material-ui.com/customization/breakpoints/)